src/tools/sdk/_linux.md

```diff
@@ -0,0 +1,41 @@
+#### Install using apt-get
```

Comment:
We need some intro text before this, or else it isn't clear that you have a choice between apt-get, debian, and compiling from source. (It looks like the text is there in https://www.dartlang.org/install/linux, so it should be easy to grab. I guess formatting is the only issue. We lose a lot of navigation when we give up the page and its TOC.)